crates/runtime/src/dataconnector.rs

```diff
@@ -93,8 +93,28 @@ pub enum Error {
     },
 }
 
+#[derive(Debug, Snafu)]
+pub enum DataConnectorError {
+    #[snafu(display("Cannot connect to {dataconnector} data connector. {source}"))]
+    UnableToConnectInternal {
+        dataconnector: String,
+        source: Box<dyn std::error::Error + Send + Sync>,
+    },
+
+    #[snafu(display("Cannot connect to {dataconnector} data connector on {host}:{port}. Ensure that the host and port are correclty configured in the spicepod, and that the host is reachable."))]
```

Comment:
```suggestion
    #[snafu(display("Cannot connect to {dataconnector} on {host}:{port}. Ensure that the host and port are correclty configured in the spicepod, and that the host is reachable."))]
```

I think the data connector is redundant here.